simapp/app.go

```diff
@@ -294,7 +294,14 @@ func NewSimApp(
 
 	app.AuthzKeeper = authzkeeper.NewKeeper(keys[authzkeeper.StoreKey], appCodec, app.msgSvcRouter, app.AccountKeeper)
 
-	app.GroupKeeper = groupkeeper.NewKeeper(keys[group.StoreKey], appCodec, app.msgSvcRouter, app.AccountKeeper)
+	groupParams := &groupkeeper.KeeperParams{
+		StoreKey:          keys[group.StoreKey],
+		Cdc:               appCodec,
+		Router:            app.msgSvcRouter,
+		AccKeeper:         app.AccountKeeper,
+		MaxMetadataLength: int(255),
```

Comment:
you can probably just omit it here in simapp since this should take the default max of 255 if not provided